build/config/compiler/BUILD.gn

```diff
@@ -761,6 +761,8 @@ config("no_chromium_code") {
 config("rtti") {
   if (is_win) {
     cflags_cc = [ "/GR" ]
+  } else {
+    cflags_cc = [ "-frtti" ]
```

Comment:
Below for no-rtti it gets applied to both cflags_cc and cflags_objcc. Should we do the same here?

Reply:
Yeah, we should just do that here too so someone using this config doesn't get surprised when it doesn't work as expected on ObjC++